### PR TITLE
Added GOVUK styling to 'Show my name publicly'

### PR DESCRIPTION
**Pull request by lucascumsille**
Fixes: https://github.com/mysociety/societyworks/issues/3825
Part of this milestone: https://github.com/mysociety/societyworks/milestone/99

Screenshot 2023-08-24 at 11 52 45

Screenshot 2023-08-24 at 11 52 24

[Skip changelog]

### CODE REVIEW

File: templates/web/base/header.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
         [% INCLUDE 'header/css.html' %]
         [% INCLUDE 'common_header_tags.html' %]
         [% TRY %][% PROCESS 'header_extra.html' %][% CATCH file %][% END %]
+        <link rel="stylesheet" href="/vendor/govuk-frontend/forms.css">
```
Comment (dracos) [inline templates/web/base/header.html:25]: Could this be in header/css.html (two templates), and then all the other includes of it in various header_extra.html etc can be removed.

**Review by dracos — CHANGES_REQUESTED** [review]
This one seems the smallest so thought might as well use this to get the forms.css in core properly and everywhere?

**Comment by lucascumsille** [comment]
@dracos Thanks for the feedback. I just pushed a commit that adds the GOVUK stylesheet to core and then I removed it from all the `header_extra`

**Comment by dracos** [comment]
Thanks. After some more investigation of this, we can import this in base.scss, replacing the radios.css that is already there (which is a subset of this CSS). I've force-pushed this branch with that change, let me know that it's still okay for you.

(Both files are being generated from https://github.com/mysociety/fixmystreet.com/tree/master/css if we ever need to make changes at source.)

**Comment by lucascumsille** [comment]
> Thanks. After some more investigation of this, we can import this in base.scss, replacing the radios.css that is already there (which is a subset of this CSS). I've force-pushed this branch with that change, let me know that it's still okay for you.
> 
> (Both files are being generated from https://github.com/mysociety/fixmystreet.com/tree/master/css if we ever need to make changes at source.)

Thanks =) yeah it's fine with me and I can see it working.

**Pull request merged**